### PR TITLE
Documentation & Adjustments

### PR DESCRIPTION
**Pull request by RocksteadyTC**

### CODE REVIEW

**Comment by RocksteadyTC** [comment]
turtlesprinkles

File: config.json [hunk 1]
```diff
@@ -53,16 +73,21 @@
             "variancePercent": 30,
             "maxJump": 100
         },
+        /* Set difficulty on miner client side by passing <address> param with .<difficulty> postfix
+       minerd -u 4AsBy39rpUMTmgTUARGq2bFQWhDhdQNekK5v4uaLU699NPAnx9CubEJ82AkvD5ScoAZNYRwBxybayainhyThHAZWCdKmPYn.5000 */
         "fixedDiff": {
             "enabled": true,
-            "addressSeparator": "."
+            "separator": ".", // character separator between <address> and <difficulty>
```
Comment (CaptainMeatloaf) [inline config.json]: Why has this been changed? It works fine as is.

File: config.json [hunk 2]
```diff
@@ -79,54 +104,66 @@
...
-        "interval": 600,
-        "maxAddresses": 50,
-        "mixin": 2,
-        "transferFee": 1000,
-        "minPayment": 10000000000000000000,
-        "maxTransactionAmount": 0,
-        "denomination": 100000000000000
+        "interval": 600, //how often to run in seconds
+        "maxAddresses": 50, //split up payments if sending to more than this many addresses
+        "mixin": 5, //number of transactions yours is indistinguishable from
```
Comment (CaptainMeatloaf) [inline config.json]: Mixin maybe too high for current network?

File: config.json [hunk 3]
```diff
@@ -79,54 +104,66 @@
...
+        "denomination": 100000000000000  //truncate to this precision and store remainder
     },
 
+    /* Module that monitors the submitted block maturities and manages rounds. Confirmed
+   blocks mark the end of a round where workers' balances are increased in proportion
+   to their shares. */
     "blockUnlocker": {
         "enabled": true,
-        "interval": 1,
+        "interval": 10, //how often to check block statuses in seconds
```
Comment (CaptainMeatloaf) [inline config.json]: Given block times are 30s I think this might be too high

File: config.json (hunk 1 above)
Comment (RocksteadyTC) [reply config.json]: I copied this part in with the comment and didnt notice the change from the copy I had.

File: config.json (hunk 2 above)
Comment (RocksteadyTC) [reply config.json]: Seems fine, and eventually with your pool sending repeated payments of various similar sizes, it wouldn't be difficult long if it were. I'm not married to this change.

**Pull request merged**